content/docs/user-guide/gems/reference/network/http-requestor.md

```diff
@@ -25,6 +25,29 @@ BUILD_DEPENDENCIES
             3rdParty::AWSNativeSDK::Core
 ```
 
+### Create a dependency on the Gem
+
+If you need to ensure the HttpRequestor Gem is enabled before its use in a component, add a requirement in your component's `GetRequiredServices` method. For example:
+
+```cpp
+void AutomatedTestingSystemComponent::GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required)
+{
+   ...
+   required.push_back(AZ_CRC_CE("HttpRequestorService"));
+}
+```
+
+### Turn off AWS EC2 Instance Metadata Service calls
+
+The HttpRequestor gem uses the [AWS C++ SDK](https://github.com/aws/aws-sdk-cpp) under-the-hood to provide the Http(s) client. The code is configured to not require any AWS credentials or account information.
+
+However, if you are not running on AWS EC2 compute its recommended that you turn off the [AWS_EC2_METADATA_DISABLED](https://github.com/aws/aws-sdk-cpp/blob/main/aws-cpp-sdk-core/source/client/ClientConfiguration.cpp#L104) environment variable. This will prevent any reach out the AWS EC2 Instance Metadate Service (IMDS), which may occur to retrieve configuration, region and credential information. Requests to EC2 IMDS will fail on non EC2 compute leading to delays and wasted network resources.
```

Comment:
AWS branding uses "Amazon EC2" instead of "AWS EC2", so I recommend sticking to brand here.
Also, I'd recommend linking to the description of the env variable instead of the source code.
```suggestion
However, if your project is not running on Amazon EC2 compute, it's recommended that you turn off the [AWS_EC2_METADATA_DISABLED](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) environment variable. This will prevent any calls to the Amazon EC2 Instance Metadata Service (IMDS) from attempting to retrieve configuration, region, and credential information. Requests to IMDS will fail if you are not using EC2, leading to delays and wasted network resources.
```

Reply:
Technically that link is to the AWS CLI, which I didn't use as I thought it was confusing because we are not configuring the AWS CLI but the AWS C++ SDK. Unfortunately the AWS C++ SDK docs do not cover this env var doc.